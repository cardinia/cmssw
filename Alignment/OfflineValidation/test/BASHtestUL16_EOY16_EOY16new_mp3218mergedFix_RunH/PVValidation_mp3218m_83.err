27-Sep-2019 21:44:23 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2016H/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/20000/569446D3-0D12-E911-A65A-0025905A60D0.root
27-Sep-2019 21:44:26 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/data/Run2016H/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/20000/569446D3-0D12-E911-A65A-0025905A60D0.root
Begin processing the 1st record. Run 283876, Event 665426942, LumiSection 443 on stream 0 at 27-Sep-2019 21:44:38.714 CEST
Begin processing the 1001st record. Run 283876, Event 672424216, LumiSection 447 on stream 0 at 27-Sep-2019 21:48:13.488 CEST
Begin processing the 2001st record. Run 283876, Event 685511774, LumiSection 455 on stream 0 at 27-Sep-2019 21:51:46.251 CEST
Begin processing the 3001st record. Run 283876, Event 695630445, LumiSection 460 on stream 0 at 27-Sep-2019 21:55:25.377 CEST
Begin processing the 4001st record. Run 283876, Event 702742591, LumiSection 464 on stream 0 at 27-Sep-2019 21:59:10.766 CEST
Begin processing the 5001st record. Run 283876, Event 709282087, LumiSection 468 on stream 0 at 27-Sep-2019 22:02:40.610 CEST
Begin processing the 6001st record. Run 283876, Event 718811565, LumiSection 473 on stream 0 at 27-Sep-2019 22:06:16.434 CEST
Begin processing the 7001st record. Run 283876, Event 724688236, LumiSection 477 on stream 0 at 27-Sep-2019 22:10:16.039 CEST
Begin processing the 8001st record. Run 283876, Event 734034412, LumiSection 482 on stream 0 at 27-Sep-2019 22:13:50.434 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 27-Sep-2019 22:16:38 CEST  Run: 283876 Event: 739923013
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 1
%MSG
Begin processing the 9001st record. Run 283876, Event 741641754, LumiSection 486 on stream 0 at 27-Sep-2019 22:17:37.905 CEST
Begin processing the 10001st record. Run 283876, Event 748414720, LumiSection 490 on stream 0 at 27-Sep-2019 22:21:15.261 CEST
Begin processing the 11001st record. Run 283876, Event 756312021, LumiSection 495 on stream 0 at 27-Sep-2019 22:24:52.951 CEST
Begin processing the 12001st record. Run 283876, Event 765209067, LumiSection 500 on stream 0 at 27-Sep-2019 22:28:35.355 CEST
Begin processing the 13001st record. Run 283876, Event 773582031, LumiSection 505 on stream 0 at 27-Sep-2019 22:32:44.075 CEST
Begin processing the 14001st record. Run 283876, Event 781967555, LumiSection 509 on stream 0 at 27-Sep-2019 22:37:11.834 CEST
Begin processing the 15001st record. Run 283876, Event 790027608, LumiSection 514 on stream 0 at 27-Sep-2019 22:41:57.350 CEST
Begin processing the 16001st record. Run 283876, Event 797762117, LumiSection 519 on stream 0 at 27-Sep-2019 22:46:56.004 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 27-Sep-2019 22:47:33 CEST  Run: 283876 Event: 798360354
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 0
%MSG
Begin processing the 17001st record. Run 283876, Event 804989642, LumiSection 523 on stream 0 at 27-Sep-2019 22:51:48.618 CEST
Begin processing the 18001st record. Run 283876, Event 814398677, LumiSection 528 on stream 0 at 27-Sep-2019 22:56:39.675 CEST
Begin processing the 19001st record. Run 283876, Event 821025243, LumiSection 532 on stream 0 at 27-Sep-2019 23:01:00.727 CEST
Begin processing the 20001st record. Run 283876, Event 828586707, LumiSection 537 on stream 0 at 27-Sep-2019 23:05:36.794 CEST
Begin processing the 21001st record. Run 283876, Event 836920155, LumiSection 542 on stream 0 at 27-Sep-2019 23:09:55.897 CEST
Begin processing the 22001st record. Run 283876, Event 843829043, LumiSection 547 on stream 0 at 27-Sep-2019 23:14:00.513 CEST
Begin processing the 23001st record. Run 283876, Event 852638384, LumiSection 552 on stream 0 at 27-Sep-2019 23:17:32.109 CEST
Begin processing the 24001st record. Run 283876, Event 861201059, LumiSection 556 on stream 0 at 27-Sep-2019 23:21:31.265 CEST
Begin processing the 25001st record. Run 283876, Event 867733796, LumiSection 560 on stream 0 at 27-Sep-2019 23:25:40.740 CEST
Begin processing the 26001st record. Run 283876, Event 875929020, LumiSection 564 on stream 0 at 27-Sep-2019 23:29:32.478 CEST
Begin processing the 27001st record. Run 283876, Event 883538186, LumiSection 567 on stream 0 at 27-Sep-2019 23:33:18.329 CEST
Begin processing the 28001st record. Run 283876, Event 890620116, LumiSection 571 on stream 0 at 27-Sep-2019 23:37:01.050 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 27-Sep-2019 23:39:07 CEST  Run: 283876 Event: 896240009
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 965
%MSG
Begin processing the 29001st record. Run 283876, Event 900126675, LumiSection 576 on stream 0 at 27-Sep-2019 23:40:52.012 CEST
Begin processing the 30001st record. Run 283876, Event 906248680, LumiSection 579 on stream 0 at 27-Sep-2019 23:44:54.100 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 27-Sep-2019 23:48:22 CEST  Run: 283876 Event: 913957176
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 467
%MSG
Begin processing the 31001st record. Run 283876, Event 913924006, LumiSection 583 on stream 0 at 27-Sep-2019 23:48:41.488 CEST
%MSG-w OutOfBounds:  TrackRefitter:FinalTrackRefitter 27-Sep-2019 23:51:46 CEST  Run: 283876 Event: 922335713
Prob Q outside the bounds of the quality word. Defaulting to Prob=0. Prob = 1 QualityWord = 970
%MSG
Begin processing the 32001st record. Run 283876, Event 922023166, LumiSection 587 on stream 0 at 27-Sep-2019 23:52:13.579 CEST
Begin processing the 33001st record. Run 283876, Event 931539739, LumiSection 591 on stream 0 at 27-Sep-2019 23:55:55.671 CEST
Begin processing the 34001st record. Run 283876, Event 939043863, LumiSection 595 on stream 0 at 27-Sep-2019 23:59:34.633 CEST
condor_exec.exe: line 14:   102 Bus error               cmsRun PVValidation_mp3218m_283876_cfg.py
[6.352kB/6.352kB][100%][==================================================][6.352kB/s]  [6.352kB/6.352kB][100%][==================================================][6.352kB/s]  
