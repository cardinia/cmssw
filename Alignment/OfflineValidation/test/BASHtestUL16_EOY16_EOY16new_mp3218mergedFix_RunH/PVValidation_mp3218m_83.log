000 (3196555.083.000) 09/27 21:34:15 Job submitted from host: <137.138.44.75:9618?addrs=137.138.44.75-9618+[2001-1458-d00-1a--b]-9618&alias=bigbird14.cern.ch&noUDP&sock=schedd_2873_37ae_40>
...
040 (3196555.083.000) 09/27 21:44:14 Started transferring input files
	Transferring to host: <128.142.97.63:9618?addrs=128.142.97.63-9618+[2001-1458-301-87--100-144]-9618&noUDP&sock=4273_7707_24763>
...
040 (3196555.083.000) 09/27 21:44:14 Finished transferring input files
...
001 (3196555.083.000) 09/27 21:44:14 Job executing on host: <128.142.97.63:9618?addrs=128.142.97.63-9618+[2001-1458-301-87--100-144]-9618&noUDP&sock=2401_76fb_3>
...
006 (3196555.083.000) 09/27 21:44:22 Image size of job updated: 232288
	119  -  MemoryUsage of job (MB)
	121804  -  ResidentSetSize of job (KB)
...
006 (3196555.083.000) 09/27 21:49:22 Image size of job updated: 665692
	547  -  MemoryUsage of job (MB)
	559352  -  ResidentSetSize of job (KB)
...
006 (3196555.083.000) 09/27 21:54:23 Image size of job updated: 665996
	547  -  MemoryUsage of job (MB)
	559656  -  ResidentSetSize of job (KB)
...
006 (3196555.083.000) 09/27 21:59:24 Image size of job updated: 666412
	547  -  MemoryUsage of job (MB)
	560072  -  ResidentSetSize of job (KB)
...
006 (3196555.083.000) 09/27 22:04:24 Image size of job updated: 668460
	549  -  MemoryUsage of job (MB)
	562120  -  ResidentSetSize of job (KB)
...
006 (3196555.083.000) 09/27 22:29:26 Image size of job updated: 668840
	550  -  MemoryUsage of job (MB)
	562492  -  ResidentSetSize of job (KB)
...
006 (3196555.083.000) 09/27 22:44:28 Image size of job updated: 669676
	551  -  MemoryUsage of job (MB)
	563328  -  ResidentSetSize of job (KB)
...
006 (3196555.083.000) 09/27 22:54:29 Image size of job updated: 669700
	551  -  MemoryUsage of job (MB)
	563352  -  ResidentSetSize of job (KB)
...
040 (3196555.083.000) 09/28 00:02:39 Started transferring output files
...
040 (3196555.083.000) 09/28 00:02:39 Finished transferring output files
...
005 (3196555.083.000) 09/28 00:02:39 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 02:16:04, Sys 0 00:00:09  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 02:16:04, Sys 0 00:00:09  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	7227  -  Run Bytes Sent By Job
	1018  -  Run Bytes Received By Job
	7227  -  Total Bytes Sent By Job
	1018  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1           
	   Disk (KB)            :       52        1   4180464 
	   Memory (MB)          :      551     2000      2000 
...
