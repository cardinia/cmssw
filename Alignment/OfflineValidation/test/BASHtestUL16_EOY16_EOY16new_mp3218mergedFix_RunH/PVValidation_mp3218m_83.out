Job started at  Fri Sep 27 21:44:14 CEST 2019
batch dir: /pool/condor/dir_21020 release: /afs/cern.ch/cms/CAF/CMSALCA/ALCA_TRACKERALIGN/data/commonValidation/alignmentObjects/paconnor/CMSSW_10_6_0/src release base: /cvmfs/cms.cern.ch/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_0
cmsRun PVValidation_mp3218m_283876_cfg.py
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: This is DATA!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Applying TrackerSurfaceDeformations!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Running DA Algorithm!
Content of working dir is total 64K -rw-r--r--. 1 tkello zh 6.4K Sep 27 21:44 PVValidation_mp3218m_283876.root -rw-r--r--. 1 tkello zh 21K Sep 27 21:44 PVValidation_mp3218m_283876_cfg.py -rw-r--r--. 1 tkello zh 5.9K Sep 28 00:02 _condor_stderr -rw-r--r--. 1 tkello zh 522 Sep 27 21:44 _condor_stdout -rwxr-xr-x. 1 tkello zh 1018 Sep 27 21:44 condor_exec.exe -rw-------. 1 tkello zh 4.3K Sep 27 21:44 tkello.cc drwx------. 3 tkello zh 4.0K Sep 27 21:44 tmp drwx------. 3 tkello zh 4.0K Sep 27 21:44 var
Job ended at  Sat Sep 28 00:02:39 CEST 2019
